shared-module/vectorio/VectorShape.h

```diff
@@ -30,6 +30,7 @@ typedef struct {
     mp_obj_t pixel_shader;
     int16_t x;
     int16_t y;
+    bool hidden : 1;
```

Comment:
I'd suggest moving this down below current_area_dirty and omitting the `: 1`. I'm not sure what size the struct will use but I know that by placing it here, you are unaligning the pointer that follows this and likely leaving a bunch of empty space.

Reply:
Thank you. Changed that in the latest commit.